packages/mobile/src/components/details-tile/DetailsTileActionButtons.tsx

```diff
@@ -178,16 +173,14 @@ export const DetailsTileActionButtons = ({
     <Flex direction='row' justifyContent='center' gap='xl'>
       {hideShare ? null : shareButton}
       {isOwner && !ddexApp
-        ? !isAlbum || isEditAlbumsEnabled
+        ? !isAlbum
           ? editButton
           : null
         : hideRepost
         ? null
         : repostButton}
       {isOwner || hideFavorite ? null : favoriteButton}
-      {isOwner && !isPublished && (!isAlbum || isEditAlbumsEnabled)
-        ? publishButton
-        : null}
+      {isOwner && !isPublished && !isAlbum ? publishButton : null}
```

Comment:
we want to remove `!isAlbum` here since the `(!isAlbum || isEditAlbumsEnabled)` clause is always truthy. We do want to show the publish button on albums now that the flag is permanently true

Reply:
great catch